Week1/homework/app.js

```diff
@@ -1,11 +1,178 @@
+  },
+  crime_and_punishment: {
+    title: 'Crime And Punishment',
+    author: 'Fyodor Dostoyevsky',
+    language: 'English',
+    year: 1866,
+    link: 'https://www.goodreads.com/book/show/7144.Crime_and_Punishment',
+  },
+};
+
+// *** 1.5
+
+function getInfo() {
+  for (let i = 0; i < bookTitles.length; i++) {
+    const listItem = document.createElement('li');
+    createList.appendChild(listItem);
+    const cover = document.createElement('img');
+    listItem.appendChild(cover);
+    const heading = document.createElement('h1');
+    listItem.appendChild(heading);
+    const authorName = document.createElement('h2');
+    listItem.appendChild(authorName);
+    const bookLanguage = document.createElement('h3');
+    listItem.appendChild(bookLanguage);
+    const publishDate = document.createElement('h4');
+    listItem.appendChild(publishDate);
+    const linkTo = document.createElement('h5');
+    listItem.appendChild(linkTo);
+  }
+
+  const obj = Object.values(booksInfo);
+
+  for (let i = 0; i < obj.length; i++) {
+    document.getElementsByTagName('h1')[i].innerHTML = obj[i].title;
+    document.getElementsByTagName('h2')[i].innerHTML = obj[i].author;
+    document.getElementsByTagName('h3')[i].innerHTML = obj[i].language;
+    document.getElementsByTagName('h4')[i].innerHTML = obj[i].year;
+    document.getElementsByTagName('h5')[i].innerHTML = obj[i].link;
+  }
+}
+getInfo();
+
+// *** 1.6
+
+//  css file created and attached to index.html
+
+//  since reaching bookCovers from here for setting src and alt attribute throws "lexical declaration `bookCovers' before initialization", it has moved below 1.7
+
+// *** 1.7
+
+const bookCovers = {
+  the_brothers_karamazov:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1427728126l/4934.jpg',
+  jane_eyre:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1557343311l/10210._SY475_.jpg',
+  to_kill_a_mockingbird:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1553383690l/2657.jpg',
+  war_and_peace:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1413215930l/656.jpg',
+  don_quixote:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1546112331l/3836._SY475_.jpg',
+  animal_farm:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1325861570l/170448.jpg',
+  1984: 'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1532714506l/40961427._SX318_.jpg',
+  the_little_prince:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1367545443l/157993.jpg',
+  frankenstein:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1498841231l/35031085.jpg',
+  crime_and_punishment:
+    'https://i.gr-assets.com/images/S/compressed.photo.goodreads.com/books/1382846449l/7144.jpg',
+};
+
+const source = Object.values(bookCovers);
```

Comment:
In your JavaScript object you are using the name of the book as key. You have the same items in your array: you should go over the elements in your array and use every item in the array to lookup a value in both objects.

Reply:
I'll go over Object.key issue and come up with right way